app/src/components/Pages/SettingsPage.tsx

```diff
@@ -0,0 +1,258 @@
+        <Button
+          variant="link"
+          onClick={() => {
+            logEvent({
+              eventName: 'EDIT_GOLD_SETTINGS',
+            });
+            window.open(
+              'https://billing.stripe.com/p/login/eVabK06mUcNG2oE6oo'
+            );
+          }}
+        >
+          Edit Subscription
+        </Button>
+      )}
+      {!isPremium && (
+        <Button
+          colorScheme="yellow"
+          onClick={() => {
+            logEvent({
+              eventName: 'UNLOCK_GOLD_SETTINGS',
+            });
+            window.open(STRIPE_LINK);
+          }}
+        >
+          Get Left on Read Gold
+        </Button>
+      )}
+    </Box>
+  );
+}
+
+function SettingsCard({
+  children,
+  icon,
+  title,
+  description,
+}: {
+  children: React.ReactNode;
+  icon: IconType;
+  title: string;
+  description: string;
+}) {
+  return (
+    <Box
+      bg="gray.100"
+      style={{
+        borderRadius: 16,
+        width: '45%',
+        padding: '24px 40px',
+      }}
+    >
+      <Box
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+        }}
+      >
+        <Box
+          style={{
+            marginRight: 12,
+            border: `1px ${defaultTheme.colors.gray['500']} solid`,
+            borderRadius: '50%',
+            padding: 8,
+            width: 40,
+            height: 40,
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+          boxShadow="lg"
+        >
+          <Icon
+            as={icon}
+            color="gray.500"
+            style={{ width: '80%', height: '80%' }}
+          />
+        </Box>
+        <Box>
+          <Text fontWeight="bold" fontSize="xl">
+            {title}
+          </Text>
+          <Text fontWeight="light" fontSize="sm" color="gray.600">
+            {description}
+          </Text>
+        </Box>
+      </Box>
+      <Box style={{ padding: 24 }}>{children}</Box>
+    </Box>
+  );
+}
+
+export function SettingsPage() {
+  return (
+    <Box style={{ padding: '36px' }}>
+      <Text fontSize="5xl" fontWeight="bold" style={{ marginBottom: 40 }}>
+        Settings
+      </Text>
+
+      <Wrap gap={6} spacing={30}>
+        <SettingsCard
+          title="Notifications"
+          icon={FiBell}
+          description="Enable and disable notifications"
+        >
+          <Notifications />
+        </SettingsCard>
+        <SettingsCard
+          title="Reset Application"
+          icon={FiTrash2}
+          description="Clears existing analytics."
+        >
+          <Application />
+        </SettingsCard>
+        <SettingsCard
+          title="Contact Us"
+          icon={FiMail}
+          description="Questions or Feedback?"
+        >
+          <ContactUs />
+        </SettingsCard>
+        <SettingsCard
+          title="Manage Subscription"
+          icon={FiMail}
+          description="Re-new, edit, or cancel Left on Read Gold"
```

Comment:
```suggestion
          description="Edit or cancel your Left on Read Gold Subscription"
```
At the very least, renew is one word no hyphen